config/webpack.config.js

```diff
@@ -366,11 +366,10 @@ module.exports = function (webpackEnv) {
             {
               test: [/\.bmp$/, /\.gif$/, /\.jpe?g$/, /\.png$/],
               type: 'asset',
-              parser: {
-                dataUrlCondition: {
-                  maxSize: imageInlineSizeLimit,
-                },
-              },
+              loader: require.resolve('file-loader'),
+              options: {
+                name: 'static/media/[name].[ext]'
+              }
```

Comment:
I dont think this will be required if you use svgs and not a png

Reply:
Also we could just increase the imageInlineSizeLimit to accommodate this no?

Reply:
removed the changes for now as no PNGs are used